### PR TITLE
Update focusoptions-preventscroll.json

### PR DESCRIPTION
**Pull request by JonathanWbn**
The `preventScroll` option is supported in Safari 13.1.

### CODE REVIEW

**Comment by atjn** [comment]
Welcome to Caniuse! Do you have a source or have you run any tests to confirm this? Disclosing the sources will help us verify the change :)

It seems to me that there is a passing test here:
https://wpt.fyi/results/html/interaction/focus/processing-model/preventScroll.html
But i haven't done any work confirming that it works in a real usecase.

File: features-json/focusoptions-preventscroll.json [hunk 1]
```diff
@@ -231,7 +231,7 @@
       "12":"n",
       "12.1":"n",
       "13":"n",
-      "13.1":"n",
+      "13.1":"y",
       "TP":"n"
```
Comment (atjn) [inline features-json/focusoptions-preventscroll.json:235]:
```suggestion
      "TP":"y"
```
If it works in Safari 13.1, it probably also works in Technology Preview. I'd be curious to know if it works in Safari on IOS as well.

**Comment by JonathanWbn** [comment]
Thank you for your help. This PR was not thought through and not based on any proper testing or research. I don't want to invest this time. I'd rather just close this, I'm sorry.

**Comment by atjn** [comment]
That's fair. You are always welcome to open an issue instead. There's a good chance someone else will be happy to invest the time :)

**Pull request closed without merging**